Vecherskiys/03/task3.add.html

```diff
@@ -0,0 +1,104 @@
+<!DOCTYPE html>
+	<html>
+	<head>
+		<META charset="utf-8"> 
+		<title>Формы</title>
+	</head>
+	<body>
+		<H1 align="center">Заполните анкету</H1>
+		<form name="Anketa" method="POST" action="http://sc.it-academy.by/fe/form.php">
+			<fieldset>
+				<legend>Личные данные</legend>
+					<input type="text" name="last_name" placeholder="Фамилия" pattern="[А-Яа-я]{1,25}" required autofocus><Br>
+					<input type="text" name="first_name" placeholder="Имя" pattern="[А-Яа-я]{1,20}" required><Br>
+					<input type="text" name="patronymic" placeholder="Отчество" pattern="[А-Яа-я]{1,24}" required><Br>
+				<fieldset>
+					<legend>Паспортные данные</legend>
+						<input type="text" name="seriya" placeholder="AA" pattern="[A-Z]{2}" required><Br>
+						<input type="text" name="number" placeholder="1234567" pattern="[0-9]{7}" required><Br>
+				</fieldset>
+				<fieldset>
+					<legend>Дата рождения</legend>
+						<input type="text" name="year" placeholder="2000" pattern="[0-9]{4}"><Br>
+						<select name="month" id="list">
+							<option value="0" selected>Январь</option>
+							<option value="1">Февраль</option>
+							<option value="2">Март</option>
+							<option value="2">Апрель</option>
+							<option value="2">Май</option>
+							<option value="2">Июнь</option>
+							<option value="2">Июль</option>
+							<option value="2">Август</option>
+							<option value="2">Сентябрь</option>
+							<option value="2">Октябрь</option>
+							<option value="2">Ноябрь</option>
+							<option value="2">Декабрь</option>
+						</select><br>
+						<input type="number" min="1" max="31" step="1"><br>
+				</fieldset>
+				<select name="grazdanstvo" id="list" required>
+					<option value="0" selected>РБ</option>
+					<option value="1">РФ</option>
+					<option value="2">ЛТ</option>
+				</select><br>
+				<textarea name="adress"></textarea>
+			</fieldset>
+			<fieldset>
+				<legend>Интересы</legend>
+				<fieldset>
+					<legend>Хобби</legend>
+					<label>
+						<input type="checkbox" name="hobby[]">Рыбалка
+				    </label><br>
+				    <label>
+						<input type="checkbox" name="hobby[]">Астрономия
+				    </label><br>
+				    <label>
+						<input type="checkbox" name="hobby[]">Дайвинг
+				    </label><br>
+				    <label>
+						<input type="checkbox" name="hobby[]">Марки
+					</label><br>
+				</fieldset>
+				<fieldset>
+					<legend>Домашние животные</legend>
+						<select name="list" id="list" multiple size="12">
```

Comment:
If you use multiple - then need to change name of variable list -> list[]

Reply:
done